ParkingLotException.java

```diff
@@ -0,0 +1,5 @@
+public class ParkingLotException extends Exception {
```

Comment:
this is a good start, but right now it is not very useful since it takes just a string and puts it into message. Think about how to pass back information about the exceptions as fields that are relevant to the parking lot library.

Reply:
There's a second commit in this pull request where I passed the value the user enters as a parameter to the exception method and added a field to the Exception class that will hold the illegal value to pass that information back to the user.